app/views/pages/home.html.erb

```diff
@@ -1,173 +1,182 @@
-        </p>
+        <h3 class="text-decoration-none featured-body-title">Toulouse to Bordeaux</h3>
+        <div class="featured-card-text">
+          <h5>France for X days</h5>
+          <h5 class="text-decoration-none featured-body-type">🍕 Foodie 🍕</h5>
+        </div>
       </div>
-    </div>
-
-    <div class="featured-card">
+    <% end %>
+    <%= link_to journey_path(@food_1), class:"feature-link featured-card" do %>
       <%= image_tag asset_path("Marseille.jpg"), class: "featured-card-img-top" %>
       <div class="featured-card-body">
-        <%= link_to "Marseille to Lyon", journey_path(@food_2), class:"text-decoration-none featured-body-title" %>
-        <p class="featured-card-text">
-          Seafood and the french gastronomy capital city all in a weekend, this is how you discover the finest cuisine in the world.
-        </p>
+        <h3 class="text-decoration-none featured-body-title">Marseille to Lyon</h3>
+        <div class="featured-card-text">
+          <h5>France for X days</h5>
+          <h5 class="text-decoration-none featured-body-type">🍕 Foodie 🍕</h5>
+        </div>
       </div>
-    </div>
-
-    <div class="featured-card">
+    <% end %>
+    <%= link_to journey_path(@food_1), class:"feature-link featured-card" do %>
       <%= image_tag asset_path("Strasbourg.jpg"), class: "featured-card-img-top" %>
       <div class="featured-card-body">
-        <%= link_to "Strasbourg", journey_path(@food_3), class:"text-decoration-none featured-body-title" %>
-        <p class="featured-card-text">
-          Can't decide if you prefer French food or German beer? Strasbourg is the place to combine your two love.
-        </p>
+        <h3 class="text-decoration-none featured-body-title">Strasbourg</h3>
+        <div class="featured-card-text">
+          <h5>France for X days</h5>
+          <h5 class="text-decoration-none featured-body-type">🍕 Foodie 🍕</h5>
+        </div>
       </div>
-    </div>
+    <% end %>
+  <%# close category %>
   </div>
+  <%# close main container %>
 </div>
 
+
+
 <%# displays count of users, journeys, cities %>
 <div id="user-city-trip-banner" class="home-info-container d-flex align-content-center justify-content-center">
   <div class="home-info-card">
-    <div class="home-info-segment">
+    <div class="home-info-segment city-trip-text">
           <%# <%= image_tag asset_path("c-d.svg"), width: "400", height: "300" %>
           <p class="fs-4 mt-4"> Already <strong> <%= CityJourney.count + 256 %> </strong> cities visited </p>
     </div>
-    <div class="home-info-segment middle-segment-borders">
+    <div class="home-info-segment city-trip-text middle-segment-borders">
         <%# <%= image_tag asset_path("w-d.svg"), width: "400", height: "300" %>
         <p class="fs-4 mt-4"> <strong> <%= Journey.count + 57 %> </strong> trips planned with Trips Selector ! </p>
     </div>
-    <div class="home-info-segment">
+    <div class="home-info-segment city-trip-text">
```

Comment:
No point redefining same styling, reuses home-info-segment and just adds color:$white in .city-trip-text